src/coreclr/src/pal/src/libunwind/README.md

```diff
@@ -1 +0,0 @@
-README
```

Comment:
Anything was supposed to change here?

Reply:
No, I'm not sure what was even changing that file. I wasn't opening it purposefully but every time I went to add a commit it was in the list